protocol-designer/src/components/organisms/StepSummary/getLiquidDisplay.tsx

```diff
@@ -13,7 +13,8 @@ export function getLiquidDisplay(
   liquidInfo: LiquidEntity[],
   t: any
 ): JSX.Element | null {
-  const liquidLength = liquidInfo.length
+  const filteredLiquids = liquidInfo.filter((l): l is LiquidEntity => l != null)
```

Comment:
can't this be simplified since ts already knows its a liquid entity?
```suggestion
  const filteredLiquids = liquidInfo.filter(liquidEntity => liquidEntity != null)
```

Reply:
can you leave a comment above this const saying that we are filtering out liquids that are undefined when the user deletes liquids that are in use?